kano_updater/ui/install_window.py

```diff
@@ -31,7 +32,11 @@ def __init__(self):
         apply_styling_to_screen(self.CSS_FILE)
 
         Gtk.Window.__init__(self)
-        self.fullscreen()
+        # self.fullscreen()
+        screen = Gdk.Screen.get_default()
+        width = screen.get_width()
+        height = screen.get_height()
+        self.set_size_request(width, height + 80)
```

Comment:
Yup, that was the Gtk hack to make the windows behave and listen to `set_keep_below(True)`. The `+80` value is to make sure the updater top app bar is above the screen and not showing. Will add a comment.